rust/cymbal/src/types/mod.rs

```diff
@@ -48,6 +49,8 @@ pub struct Exception {
 pub struct ErrProps {
     #[serde(rename = "$exception_list")]
     pub exception_list: Option<Vec<Exception>>, // Required from exception producers - we will not process events without this. Optional to support older clients, should eventually be removed
+    #[serde(rename = "$exception_issue_id")]
```

Comment:
I'm assuming serialization will still pass here if the `$exception_issue_id` property is missing... not just that it's allowed to be `Non`

Reply:
You'll want to add `skip_serialising_if` I think, unless you want the json to have a `null` in it. See my refactor PR for how that looks (it adds the fingerprint)